deploy/chart/templates/0000_50_olm_00-namespace.yaml

```diff
@@ -4,6 +4,7 @@ metadata:
   name: {{ .Values.namespace }}
   labels:
     openshift.io/run-level: "1"
+    openshift.io/cluster-monitoring: "true"
```

Comment:
I don't want to block the PR on this, but these labels probably shouldn't go in upstream releases

Reply:
fixed!